frontend/src/components/common/table/table.tsx

```diff
@@ -1,21 +1,42 @@
+import { ReactComponent as DeleteIcon } from 'assets/icons/trash-can-solid.svg';
 import { ReactElement } from 'react';
-import { Column, useTable } from 'react-table';
+import { CellProps, Column, useTable } from 'react-table';
 
 import styles from './styles.module.scss';
 
 type Props<Data extends Record<string, unknown>> = {
   columns: Column<Data>[];
   data: readonly Data[];
+  onClick: (userId: string) => void;
 };
 
 const Table = <Data extends Record<string, unknown>>({
   columns,
   data,
+  onClick,
```

Comment:
Not sure about passing `onClick` to `Table` component. At least because there's also edit button. Maybe `onDelete`?

Reply:
agreed